### DISCUSSION TITLE
Setting @Level from a property?

### DISCUSSION THREAD

**Discussion by nastynaz** [datalust/seq-tickets#2175] [General]
I'm using Rust's `tracing-opentelemetry` crate to ingest traces in OTLP format. This sets the level (info, warn, debug etc) in a property called `level`. How can I get `seq` to recognise this field and use it as the `@Level` property? 

Is this possible or would I have to write my own adapter to set `@l` before sending the logs?

**Comment by nblumhardt** [comment]
Hi @nastynaz, thanks for your message.

Unfortunately this would need to be done client-side, converting to Seq's HTTP/JSON format instead of OTLP.

The OTLP format defines the allowed values for span statuses (`OK` and `ERROR`, I believe) and maps them to numbers, so there won't be a way to customize how this value is transmitted using OTLP.

I'll drop you a line here if we come up with any other ideas. Hope this helps!
